### PR TITLE
[build.sh] Run jazzy to generate RealmSwift docs

### LINKED ISSUE #1670: Add swift docs generation to build.sh

### PR DESCRIPTION
**Pull request by segiddins**
Closes #1670.

What other jazzy flags do we want to pass?

\c @jpsim

### CODE REVIEW

File: scripts/build-docs.sh [hunk 1]
```diff
@@ -28,10 +29,24 @@ appledoc \
...
+  --author_url "http://realm.io" \
+  --clean \
+  --github_url https://github.com/realm/realm-cocoa \
+  --github-file-prefix https://github.com/realm/realm-cocoa/tree/v${realm_version} \
+  --min-acl public \
+  --module RealmSwift \
+  --module-version ${realm_version} \
+  --output "${SRCROOT}/docs/swift_output" \
+  --root-url http://realm.io/docs/swift/api/ \
+  --skip-undocumented \
```
Comment (jpsim) [inline scripts/build-docs.sh]: please remove this

**Comment by jpsim** [comment]
:+1:

File: scripts/build-docs.sh (hunk 1 above)
Comment (segiddins) [inline scripts/build-docs.sh]: done

File: scripts/build-docs.sh [hunk 2]
```diff
@@ -28,10 +29,23 @@ appledoc \
...
 rm -rf ${SRCROOT}/docs/output/${realm_version}
 mv ${SRCROOT}/docs/html ${SRCROOT}/docs/output/${realm_version}
+
+${jazzy} \
+  --author Realm \
+  --author_url "http://realm.io" \
+  --clean \
+  --github_url https://github.com/realm/realm-cocoa \
+  --github-file-prefix https://github.com/realm/realm-cocoa/tree/v${realm_version} \
+  --min-acl public \
```
Comment (jpsim) [inline scripts/build-docs.sh]: This is now Jazzy's default, so you can remove this

**Comment by jpsim** [comment]
:+1:

**Pull request merged**